crates/storage/Cargo.toml

```diff
@@ -23,6 +23,7 @@ strata-storage-common.workspace = true
 
 anyhow.workspace = true
 async-trait.workspace = true
+borsh.workspace = true
```

Comment:
why?

Reply:
The mmr handle expects the type(leaf entry) to be borsh serializable. This is explicitly required for mmr manager because all other managers are based on concrete types which have already implemented Borsh de/serialization but for mmr entries we store them as raw bytes in the database and need to be serialized in the manager layer.